src/java/org/apache/cassandra/streaming/management/SessionInfoCompositeData.java

```diff
@@ -151,7 +151,7 @@ public StreamSummary apply(CompositeData input)
                                            connecting,
                                            fromArrayOfCompositeData((CompositeData[]) values[5], toStreamSummary),
                                            fromArrayOfCompositeData((CompositeData[]) values[6], toStreamSummary),
-                                           StreamSession.State.valueOf((String) values[7]));
+                                           StreamSession.State.valueOf((String) values[7]), null);
```

Comment:
please document why you are using `null` here.  Its fine to say its null to maintain backwards compatibility, but its still best to say this so the next reader doesn't question why its null